dists/emscripten/build.sh

```diff
@@ -0,0 +1,207 @@
+
+  games=true
+  if [ "$games" = true ]; then
+    mkdir -p ./dists/emscripten/games/
+    cd "${ROOT_FOLDER}/dists/emscripten/games/"
+    wget -nc https://downloads.scummvm.org/frs/demos/scumm/ft-dos-demo-en.zip
+    unzip -n ft-dos-demo-en -d "${ROOT_FOLDER}/build-emscripten/games/ft-dos-demo-en/"
+    wget -nc https://downloads.scummvm.org/frs/demos/mohawk/myst-win-demo-en.zip
+    unzip -n myst-win-demo-en.zip -d "${ROOT_FOLDER}/build-emscripten/games/myst-win-demo-en/"
+    wget -nc https://downloads.scummvm.org/frs/demos/mohawk/riven-win-demo-en.zip
+    unzip -n riven-win-demo-en.zip -d "${ROOT_FOLDER}/build-emscripten/games/riven-win-demo-en/" -x DXSETUP/* -x QTWSETUP/*
+    wget -nc https://downloads.scummvm.org/frs/demos/sword2/sword2-win-demo-en.zip
+    unzip -n sword2-win-demo-en.zip -d "${ROOT_FOLDER}/build-emscripten/games/sword2-win-demo-en/"
+    wget -nc https://downloads.scummvm.org/frs/demos/scumm/comi-win-large-demo-en.zip
+    unzip -n comi-win-large-demo-en.zip -d "${ROOT_FOLDER}/build-emscripten/games/comi-win-large-demo-en/"
+    wget -nc https://downloads.scummvm.org/frs/demos/grim/emi-win-demo-en.zip
+    unzip -n emi-win-demo-en.zip -d "${ROOT_FOLDER}/build-emscripten/games/emi-win-demo-en/"
+    wget -nc https://downloads.scummvm.org/frs/demos/grim/grim-win-demo2-en.zip
+    unzip -n grim-win-demo2-en.zip -d "${ROOT_FOLDER}/build-emscripten/games/grim-win-demo2-en/"
+    wget -nc https://downloads.scummvm.org/frs/demos/agos/feeble-dos-ni-demo-en.zip
+    unzip -n feeble-dos-ni-demo-en.zip -d "${ROOT_FOLDER}/build-emscripten/games/feeble-dos-ni-demo-en/"
+    wget -nc https://downloads.scummvm.org/frs/demos/sci/lsl7-dos-demo-en.zip
+    unzip -n lsl7-dos-demo-en.zip -d "${ROOT_FOLDER}/build-emscripten/games/lsl7-dos-demo-en/"
+    wget -nc https://downloads.scummvm.org/frs/demos/sci/phantasmagoria-dos-win-demo-en.zip
+    unzip -n phantasmagoria-dos-win-demo-en -d "${ROOT_FOLDER}/build-emscripten/games/phantasmagoria-dos-win-demo-en/"
+    wget -nc https://downloads.scummvm.org/frs/extras/Beneath%20a%20Steel%20Sky/BASS-Floppy-1.3.zip
+    unzip -n BASS-Floppy-1.3.zip -d "${ROOT_FOLDER}/build-emscripten/games/bass-floppy/"
+    wget -nc https://downloads.scummvm.org/frs/extras/Drascula_%20The%20Vampire%20Strikes%20Back/drascula-1.0.zip
+    unzip -n drascula-1.0.zip -d "${ROOT_FOLDER}/build-emscripten/games/drascula/"
+    wget -nc https://downloads.scummvm.org/frs/extras/Drascula_%20The%20Vampire%20Strikes%20Back/drascula-audio-mp3-2.0.zip
+    unzip -n drascula-audio-mp3-2.0.zip -d "${ROOT_FOLDER}/build-emscripten/games/drascula/"
+    wget -nc https://downloads.scummvm.org/frs/extras/Flight%20of%20the%20Amazon%20Queen/FOTAQ_Floppy.zip
+    unzip -n FOTAQ_Floppy.zip -d "${ROOT_FOLDER}/build-emscripten/games/fotaq-floppy/"
+    wget -nc https://downloads.scummvm.org/frs/extras/Lure%20of%20the%20Temptress/lure-1.1.zip
+    unzip -n lure-1.1.zip -d "${ROOT_FOLDER}/build-emscripten/games/lure/"
```

Comment:
Would you perhaps also need a pre-made scummvm.ini? Or how are you handling the initial run?

Reply:
Yes, a pre-made scummvm.ini would make the most sense. Hows the best way to go about that? I currently inject a default into the browsers local storage if there's nothing there. That's a bit of a hack though if there's a concept of a default ini file.

Reply:
Not sure if this should really be the way to go - maybe we should think about adding a download button for the freeware games or/and demos to the launcher. That way every scummvm installation could benefit from it. And the best thing would be that the emscripten port would automatically get new releases to its launcher without us having to maintain a scummvm.ini or the download urls like here.

it would imo only be a temporary solution.

Reply:
The great thing with a web port is that such a download wouldn't be necessary. All files can be downloaded on request (and to a certain point should be - browsers have their own caching system and allow web apps to store limited amounts of data persistently).

The handling of this is definitely something I'd like to improve a lot on.

Reply:
I'll mark this as resolved since I decided this is probably better solved later then in this initial PR (since it will also require some thinking about how demos are distributed/packed)